Fix a validate command that failed when the playbook's value is not str (#2465)
demisto/demisto-sdk: CHANGELOG.md

## Unreleased
* Fixed an issue where **generate-docs** with `-c` argument updated sections of the incorrect commands.
* Added IF113 error code to **ALLOWED_IGNORE_ERRORS**.

> * Fixed an issue where the **validate** failed on playbooks containing input values that is not str type.

Reviewer: ```suggestion
* Fixed an issue where **validate** failed on playbooks with non-string input values.
```
